### ISSUE TITLE
Create keyboard shortcuts for Spacer Presets

### ISSUE THREAD

**Issue by MadtownLems** [WordPress/gutenberg#65489]
## What problem does this address?

Currently, it's tedious to insert a Spacer Block with the keyboard (/spacer) and then rely on the mouse to choose a preset.

## What is your proposed solution?

It would be cool to introduce keyboard shortcuts for Spacer blocks that immediately insert the spacer block to the preset value, similar to how one can use /h2 or /h3 to insert a heading set at the correct level.

For example, /spacer2 could insert a Spacer with the preset value of 2 already selected.

**Comment by Mayank-Tripathi32** [comment]
I agree that adding preset options via keyboard shortcuts would improve accessibility. I'll reach out in the slack channels to gather additional input before moving forward with a potential solution.

**Comment by Mayank-Tripathi32** [comment]
cc: @WordPress/gutenberg-components @WordPress/accessibility

Any input on this would be greatly appreciated. Thanks
